packages/docregistry/src/default.ts

```diff
@@ -102,6 +102,10 @@ export class DocumentModel
     return this._defaultLang;
   }
 
+  get collaborative(): boolean {
```

Comment:
```suggestion
  /**
   * Whether the model is collaborative or not.
   */
  get collaborative(): boolean {
```

The returned value should be the value of the to-be-added `collaborationEnabled` argument to be passed to the constructor.